### ISSUE TITLE
Programmical plugin registration

### ISSUE THREAD

**Issue by XakepSDK** [astei/velocity#64]
Add a way to register plugin through API. I need this for bungee api proxying.

**Comment by astei** [comment]
I'm unlikely to grant this request, for a few major reasons:

* Velocity is not intended to provide full compatibility for the BungeeCord API. It has a very different design philosophy.
* Velocity intentionally does not support dynamic plugin enabling and disabling. This approach has proven to be extremely prone to breakage (just look at everyone who admonishes the use of /reload on Bukkit). This is similar to what Sponge does and it's an approach I consider to be better overall.

In conclusion, you'd be better off keeping the Velocity and "fake Bungee" worlds separate, if you _really_ wanted to undertake this project.

Even better, why not write some native Velocity plugins? We've already got a few major plugins ported to Velocity (ServerListPlus, LuckPerms, and soon NuVotifier), and these plugins help form the core of most peoples' servers. We could use a more complete suite.
